packages/experiments/src/slots/examples/Slots.Recomposition.Example.tsx

```diff
@@ -0,0 +1,73 @@
+/** @jsx withSlots */
```

Comment:
withSlots no longer needed?

Reply:
If there are no longer any closures (which I don't see) then we can finally get rid of withSlots. 🎉 Should also check the React hierarchy and make sure slotted components are not rerendering on every pass, as well.

Reply:
I _think_ it is still needed, I remember the page throwing me an error in the new example I added when it didn't have `withSlots` at the top.

Reply:
Oh, I bet it's needed for `Stack`. Assuming `composed` is not using `slots.ts` (which I don't see) then none of the components you've created should be throwing the error.

Reply:
Scratch that, it's definitely not needed for `Stack`. It should only be needed in files that render the old-style slots. Can you check again?

Reply:
For sure, I'll check again.

Reply:
By the way, I think `composed` is still using `slots.ts`, especifically the functions `createFactory` and `getSlots`. Just wanted to point that out.

Reply:
Ok, then `getSlots` is what's throwing the error. I wasn't sure if we'd be able to get rid of the closures yet, but as David's mentioned, that will change how we use slots and require the view to spread slotprops manually on each slot.